src/Module 2/Rates.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Module_2;
+
+internal class Rates
+{
+    public int Cur_ID { get; set; }
```

Comment:
вы преобразуете string в объект этого класса с помощью `System.Text.Json.JsonSerializer`

С# свойство можно назвать согласно код-конвенции языка, а правильная десериализация будет обеспена атрибутом:
```suggestion
        [JsonPropertyName("Cur_ID ")]
        public int CurId { get; set; }
```